src/tests/travis_mysql.cfg

```diff
@@ -1,5 +1,5 @@
 [database]
-backend=mysql
+backend=sqlite3
```

Comment:
Why the filename is "my_sql.cfg" and the content is "sqlite3"?

We are removing MySQL support, then this file should be deleted.

Reply:
I have deleted the file. I don't know how i missed it.